# HG changeset patch
# User alexander-wilms
# Date 1522515170 -7200
#      Sat Mar 31 18:52:50 2018 +0200
# Branch stable
# Node ID b5b83fcbb4989406bfd9deb8f19998692f4d1d93
# Parent  22bc3bcabba46ec5fa36e5516950be92a5310028
* file-editor-tab.cc: Fix typo: chose -> choose

diff --git a/libgui/src/m-editor/file-editor-tab.cc b/libgui/src/m-editor/file-editor-tab.cc
--- a/libgui/src/m-editor/file-editor-tab.cc
+++ b/libgui/src/m-editor/file-editor-tab.cc
@@ -2251,17 +2251,17 @@ namespace octave
 
     if (! codec->canEncode (_edit_area->text ()))
       {
         int ans = QMessageBox::warning (nullptr,
                                         tr ("Octave Editor"),
                                         tr ("The current editor contents can not be encoded\n"
                                             "with the selected encoding %1.\n"
                                             "Using it will result in data loss!\n\n"
-                                            "Do you want to chose another encoding?").arg (_encoding),
+                                            "Do you want to choose another encoding?").arg (_encoding),
                                         QMessageBox::Yes | QMessageBox::No, QMessageBox::Yes);
 
         if (ans == QMessageBox::Yes)
           return true;
       }
 
     return false;
   }
